plugin/trino-hive/src/test/java/io/trino/plugin/hive/parquet/AbstractTestParquetReader.java

```diff
@@ -1425,7 +1425,7 @@ public void testNewAvroArray()
 
     /**
      * Test reading various arrays schemas compatible with spec
-     * https://github.com/apache/parquet-format/blob/master/LogicalTypes.md#lists
+     * <a href="https://github.com/apache/parquet-format/blob/master/LogicalTypes.md#lists">here</a>
```

Comment:
Not necessary

Reply:
I agree that `<a href...` is the correct way to embed links in javadoc.
Note that tests'  javadocs are read within IDE and then plain link (without `<a>`) is simply more readable.
Javadocs for production code, especially for SPI, may be published as web pages and they need to be properly formed. They should use `<a>` tags for links.

Reply:
reverted the java doc changes.